data/sql/updates/world/2016_08_19_00.sql

```diff
@@ -0,0 +1,44 @@
+/*
+Reference some issues: one problem #96
+Fixing worldserver.exe errors
+
+Server: Local (Repo AzerothCore)
+Last commit hash tested: 3761e9d8433f4d031b3477b787134a1d868a8971
+O.S.: Windows 10
+Compiler: MS Visual Studio 2013
+*/
```

Comment:
those comments should be placed into the PR description, not into the code itself